symplyphysics/laws/electricity/electromotive_force_induced_in_rotating_rod.py

```diff
@@ -72,6 +84,31 @@
         \mathcal{E} = \frac{1}{2} B \omega l^2
 """
 
+# Derive law
+
+_time = _emf_law.time
+
+_angle = _angular_speed_law.angular_distance(_time)
+
+# Area of the circular sector
+_loop_area = length**2 * _angle / 2
+
+_magnetic_flux = _magnetic_flux_law.law.rhs.subs({
+    _magnetic_flux_law.induction: magnetic_flux_density,
+    _magnetic_flux_law.area: _loop_area,
+    _magnetic_flux_law.angle: 0,  # rod is orthogonal to magnetic field
+})
+
+_emf = _emf_law.law.rhs.subs({
+    _emf_law.coil_turn_count: 1,
```

Comment:
But there is no coil.

Reply:
You can think of a contour as a coil with only one turn. The formulas of EMF for a coil and a contour only differ in the presence of the number of turns the contour makes around itself, I doubt it's needed to make a separate law for that.

Reply:
Write some explanation, why there is any contour in a rod and why it can be replaced for a coil.